tests/integration/IntegrationSpec.hs

```diff
@@ -60,12 +62,14 @@ main = runSimpleApp $ do
                      <> fromString (takeFileName next)
               res <- test next
               loop (idx + 1) rest' (res <> accum)
-
+    
     loop (1 :: Int) (Set.toList tests) mempty
-
+  finalTime <- getCurrentTime
   let (successes, failures) = partition ((== ExitSuccess) . snd)
                             $ Map.toList results
-
+  let timeDiff = diffUTCTime finalTime startTime
+  let timeDiffStr = formatTime defaultTimeLocale "%h:%m:%s" timeDiff
+  logInfo (fromString ("Integration tests ran in : " ++ timeDiffStr))
```

Comment:
The `OverloadedStrings` means that you do not have to apply the `fromString` to the literal:

`logInfo $ "Integration tests ran in : " <> fromString timeDiffStr`